build_library/disk_util

```diff
@@ -637,13 +642,25 @@ def Mount(options):
   for mount in sorted(mounts, key=len):
     DoMount(mounts[mount])
 
+
 def Umount(options):
-  """Unmount the given path.
```

Comment:
Once the mount command is split to just mount to a specific directory instead of interpreting `part['mount']` this function can be left as-is